src/logger.ts

```diff
@@ -43,32 +44,66 @@ export class Logger implements ILogger {
     this.config.accessToken = token;
   }
 
-  log (message: string | Error, details?: any, skipServer = false): void {
-    this.logMessage('log', message, details, skipServer);
+  log (message: string | Error, details?: any, opts?: ILogMessageOptions): void {
```

Comment:
Do you think it wouldn't be worth it to make this backwards compatible so we wouldn't have to do a major version bump? 

Thinking like: 
``` ts
log (message: string | Error, details?: any, opts?: ILogMessageOptions | boolean): void;

// something like (logic isn't fully fleshed out)
if (typeof opts === 'boolean' || !opts) {
  const defaultOptions = {
    skipDefaultFormatter: false,
    skipServer: typeof opts === 'boolean' ? opts : false,
    skipSecondaryLogger: false
   } 
}
```
Then we could have some kind of "default" option that we pass in. Just a thought since we did just do a major version release of this. It would make the migration to formatters less painful.